### PR TITLE
feat: pyvw label accessors for multilabel

### PR DESCRIPTION
**Pull request by bassmang**

### CODE REVIEW

File: python/pylibvw.cc [hunk 1]
```diff
@@ -1008,6 +1008,13 @@ py::object ex_get_ccb_explicitly_included_actions(example_ptr ec)
   return varray_to_pylist(label.explicit_included_actions);
 }
 
+size_t ex_get_multilabel_num_labels(example_ptr ec) { return ec->l.multilabels.label_v.size(); }
```
Comment (jackgerrits) [inline python/pylibvw.cc:1011]: It might be simpler to just return a py::list for this one?

Comment (bassmang) [reply python/pylibvw.cc]: Sure that makes it a bit simpler

File: python/vowpalwabbit/pyvw.py [hunk 2]
```diff
@@ -1468,6 +1468,31 @@ def __str__(self):
         )
 
 
+class CBContinuousLabelElement:
```
Comment (jackgerrits) [inline python/vowpalwabbit/pyvw.py:1471]: Looks like this was added by mistake?

Comment (bassmang) [reply python/vowpalwabbit/pyvw.py]: good catch!

**Review by jackgerrits — APPROVED** [review]

**Pull request merged**